src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/IntervenerService.java

```diff
@@ -112,10 +115,9 @@ private void checkIfIntervenerSolicitorDetailsChanged(IntervenerWrapper interven
         IntervenerWrapper beforeIntv = intervenerWrapper.getIntervenerWrapperFromCaseData(beforeData);
 
         if (ObjectUtils.isNotEmpty(beforeIntv)
-            && beforeIntv.getIntervenerRepresented() != null
-            && beforeIntv.getIntervenerRepresented().equals(YesOrNo.YES)) {
+            && YesOrNo.YES.equals(beforeIntv.getIntervenerRepresented())) {
             String beforeOrgId = beforeIntv.getIntervenerOrganisation().getOrganisation().getOrganisationID();
-            if (!beforeOrgId.equals(orgId) || !beforeIntv.getIntervenerSolEmail().equals(email)) {
+            if (ObjectUtils.notEqual(beforeData, orgId) || !beforeIntv.getIntervenerSolEmail().equals(email)) {
```

Comment:
Is this an error?  

beforeData is an instance of FinremCaseData.  So I can't see these would ever be equal?

Reply:
good catch